{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py

```diff
@@ -1,9 +1,15 @@
 # Django settings for project project.
-from os import environ
-from django.core.exceptions import ImproperlyConfigured
+from environ import Env, Path
 import pathlib
+from django.core.exceptions import ImproperlyConfigured
+
+env = Env()
 
 
+Env.read_env('.env')
+DEBUG = env('DEBUG')
```

Comment:
I'd prefer if `DEBUG` is not managed by env var. It's too dangerous. Let's set it to `False` in the base and then `True` in local.py only.

I think we can say the same about `SSLIFY`, not sure.